test/models/journeyDomain/RouteDetailsDomainSpec.scala

```diff
@@ -38,6 +39,45 @@ class RouteDetailsDomainSpec extends SpecBase with ScalaCheckPropertyChecks with
 
   "RouteDetailsDomain" - {
 
+    "specificCircumstanceIndicatorReader" - {
+      "can be parsed from UserAnswers" - {
+        "when security is 2 or 3 " in {
+          forAll(Gen.oneOf(EntrySummaryDeclarationSecurityDetails, ExitSummaryDeclarationSecurityDetails).sample.value,
```

Comment:
I think these may be incorrect. I was confused with them on my previous ticket. It looks like 2 = Exit Summary Declaration (EXS) and 3 = Both entry and exit summary declarations (ENS and EXS)

Reply:
Oh yes good spot ^ Security codes are 0, 1, 2 and 3 (rather than 1 to 4)